frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -312,11 +312,41 @@ const determineAvailableFilters = (initialFilters, items, filterGroups) => {
   return filters;
 };
 
-const getActiveFilters = (keywordFilter, groupFilters, activeFilters) => {
+const getActiveFilters = (
+  keywordFilter,
+  groupFilters,
+  activeFilters,
+  storeFilterKey = null,
+  filterRetentionPreference = null,
+) => {
   activeFilters.keyword.value = keywordFilter || '';
   activeFilters.keyword.active = !!keywordFilter;
 
+  const userFilters = storeFilterKey ? localStorage.getItem(storeFilterKey) : null;
+  if (userFilters) {
+    try {
+      const lastFilters = JSON.parse(userFilters);
+      if (lastFilters) {
+        if (filterRetentionPreference) {
+          _.each(filterRetentionPreference, (filterGroup) => {
+            if (!groupFilters || !groupFilters[filterGroup]) {
+              if (lastFilters[filterGroup]) {
+                activeFilters[filterGroup] = lastFilters[filterGroup];
+              }
+            }
+          });
+        }
+      }
+    } catch (e) {
```

Comment:
Do you get an "unused variable" error here?
```suggestion
    } catch () {
```

Reply:
catch throws error without the argument.